pkg/controller/node/nodecontroller.go

```diff
@@ -132,9 +133,11 @@ type NodeController struct {
 	// allocate/recycle CIDRs for node if allocateNodeCIDRs == true
 	cidrAllocator CIDRAllocator
 
-	forcefullyDeletePod       func(*api.Pod) error
-	nodeExistsInCloudProvider func(string) (bool, error)
-	computeZoneStateFunc      func(nodeConditions []*api.NodeCondition) zoneState
+	forcefullyDeletePod        func(*api.Pod) error
+	nodeExistsInCloudProvider  func(string) (bool, error)
+	computeZoneStateFunc       func(nodeConditions []*api.NodeCondition) zoneState
+	enterPartialDisruptionFunc func(nodeNum int, defaultQPS float32) float32
+	enterFullDisruptionFunc    func(nodeNum int, defaultQPS float32) float32
```

Comment:
After more looking into the code, I think that the signatures of these functions should be:

func(defaultQPS float32) RateLimitedTimedQueue